restlight-core/src/main/java/io/esastack/restlight/core/resolver/RequestEntity.java

```diff
@@ -49,13 +49,6 @@
      *
      * @param ins ins
      */
-    void inputStream(InputStream ins);
-
-    /**
-     * Obtains the {@link HttpRequest}.
-     *
-     * @return request
-     */
-    HttpRequest request();
```

Comment:
Why delete the method named request?

Reply:
the request is unnecessary for RequestEntity, it's just used in `getMediaType`. and can be provided by other channel.